### ISSUE TITLE
Unable to enter SMS verification code for authentication process

### ISSUE THREAD

**Issue by forrby** [homebridge-simplisafe3/homebridge-simplisafe3#369]
### Description

Hello,

I am completely new to all of this. Just trying to get Simplisafe to work with homekit. So I installed Homebridge on my Macbook, and downloaded the simplisafe3 plugin. I'm trying to start the authentication process. After entering my email, password and pressing "Start Authentication" I get stuck with a message that says "Attempting to login with credentials..." When I press "start authentication" I do get a SMS with a verification code from
Simplisafe buy I have no idea where to enter this code. Any help would enourmously appreciated!! (And sorry if this is a super dumb question...)

### Expected behavior

In the instructions it says, "During the process you will either have to verify via email or enter an SMS vertification code (same as logging into SimpliSafe website)" but I do not have an option to verify the SMS verification code.

### Steps to reproduce

1. Enter email
2. Enter password
3. Press "Start Authentication" 
4. I get stuck with a message that says "Attempting to login with credentials..." 

### Homebridge logs

```bash
[11/17/2022, 7:01:51 PM] Homebridge v1.5.1 (HAP v0.10.4) (Homebridge A5FB) is running on port 51377.
[11/17/2022, 7:02:32 PM] [Homebridge UI] [homebridge-simplisafe3] Incoming Request: /credentialsExist
[11/17/2022, 7:02:32 PM] [Homebridge UI] [homebridge-simplisafe3] Incoming Request: /nLoginSteps
[11/17/2022, 7:02:36 PM] [Homebridge UI] [homebridge-simplisafe3] Incoming Request: /initiateLoginAndAuth
[11/17/2022, 7:02:36 PM] [Homebridge UI] [homebridge-simplisafe3] Loading login auth url...
[11/17/2022, 7:02:40 PM] [Homebridge UI] [homebridge-simplisafe3] Attempting to login with credentials...
[11/17/2022, 7:10:48 PM] [Homebridge UI] [homebridge-simplisafe3] Terminating child process...
[11/17/2022, 7:10:48 PM] [Homebridge UI] [homebridge-simplisafe3] Child process ended
[11/17/2022, 7:12:14 PM] [Homebridge UI] [homebridge-simplisafe3] Incoming Request: /credentialsExist
[11/17/2022, 7:12:14 PM] [Homebridge UI] [homebridge-simplisafe3] Incoming Request: /nLoginSteps
[11/17/2022, 7:12:46 PM] [Homebridge UI] [homebridge-simplisafe3] Incoming Request: /initiateLoginAndAuth
[11/17/2022, 7:12:46 PM] [Homebridge UI] [homebridge-simplisafe3] Loading login auth url...
[11/17/2022, 7:12:46 PM] [Homebridge UI] [homebridge-simplisafe3] Attempting to login with credentials...
[11/17/2022, 7:13:06 PM] [Homebridge UI] [homebridge-simplisafe3] Terminating child process...
[11/17/2022, 7:13:06 PM] [Homebridge UI] [homebridge-simplisafe3] Child process ended
[11/17/2022, 7:20:17 PM] [Homebridge UI] [homebridge-simplisafe3] Incoming Request: /credentialsExist
[11/17/2022, 7:20:17 PM] [Homebridge UI] [homebridge-simplisafe3] Incoming Request: /nLoginSteps
[11/17/2022, 7:20:26 PM] [Homebridge UI] [homebridge-simplisafe3] Incoming Request: /initiateLoginAndAuth
[11/17/2022, 7:20:26 PM] [Homebridge UI] [homebridge-simplisafe3] Loading login auth url...
[11/17/2022, 7:20:26 PM] [Homebridge UI] [homebridge-simplisafe3] Attempting to login with credentials...
[11/17/2022, 7:24:48 PM] [Homebridge UI] [homebridge-simplisafe3] Terminating child process...
[11/17/2022, 7:24:48 PM] [Homebridge UI] [homebridge-simplisafe3] Child process ended
```

### Homebridge config.json contents

```json
Don't know how to do this.... I'm sorry... I feel like this may be an easy fix I just am not familiar with all of this :(
```

### Screenshots

![Screen Shot 2022-11-17 at 7 24 53 PM](https://user-images.githubusercontent.com/118493783/202596292-983f5853-72d8-4784-a377-3c20a3762c0b.png)
![Screen Shot 2022-11-17 at 7 20 29 PM](https://user-images.githubusercontent.com/118493783/202596295-98254cff-3d3f-4983-bee2-ce83174019cd.png)
![Screen Shot 2022-11-17 at 7 20 23 PM](https://user-images.githubusercontent.com/118493783/202596297-35d2ceff-ec75-4302-95a8-626f881b621f.png)

### Plugin version

v1.10.10

### Homebridge version

v4.50.2

### Node.js version

v16.16.0

### Host OS

macOS

**Comment by Trist0ne** [comment]
Seeing an identical issue, unable to login. Tried fully removing and reinstalling, didn't help. Command line authentication is not working either

**Comment by forrby** [comment]
Yes I also tried with the command line. Initially when I tried with the command line it actually said that it was authenticated in the terminal but I never receivd and SMS verification code and on the homebridge UI nothing happened. Trying to do it with the command line now to take a screenshot it did not work and im getting "Authentication error: Request failed with status code 400" Screen shots attached. 
![Screen Shot 2022-11-17 at 10 51 09 PM](https://user-images.githubusercontent.com/118493783/202620113-b0c9651d-08cf-4ac0-a282-670c20d7a4e2.png)
